debezium-core/src/main/java/io/debezium/transforms/converters/ByteBufferConverter.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.transforms.converters;
```

Comment:
Could you move it to `io.debezium.converters`? It's not really related to transforms.